### PR TITLE
docs(repo): Add getting started section to README

### PR DESCRIPTION
**Pull request by johanndev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +36,31 @@ Right now, the state of it is as follows:
...
   - Template Files
   - Component Style Files
+
+## Getting started
+```bash
+# Create a new nx workspace
+npx create-nx-workspace ng-rspack-test
+# Choose options:
+# - Stack: None
+# - Package-based Monorepo
```
Comment (Coly010) [inline README.md:46]:
```suggestion
# - Integrated Monorepo
```

File: README.md [hunk 2]
```diff
@@ -36,3 +36,31 @@ Right now, the state of it is as follows:
...
+# - CI: Do it later
+# - Remote caching: Up to you
+
+# Install the ng-rspack-build package
+npm install @ng-rspack/nx
+
+# Change into project directory and run the app generator
+cd ng-rspack-test
```
Comment (Coly010) [inline README.md:54]: Move this before the the `npm install @ng-rspack/nx`

**Review by Coly010 — CHANGES_REQUESTED** [review]
Hey! Thanks for the contribution, some small changes then this will be good to merge.

File: README.md (hunk 2 above)
Comment (johanndev) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (johanndev) [reply README.md]: Done

**Review by Coly010 — APPROVED** [review]

**Comment by Coly010** [comment]
Awesome! thank you for this 🎉

**Comment by johanndev** [comment]
Awesome, thank you for this project :) 

I'm really excited to get to use rspack with Angular!

**Pull request merged**